.github/workflows/main.yaml

```diff
@@ -0,0 +1,20 @@
+name: buildkite-pipeline
+on: [push]
+
+jobs:
+  trigger:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
```

Comment:
We might want to consider version pinning this - both the docs and the README for actions/checkout pin to v2.